### DISCUSSION TITLE
create a new repository

### DISCUSSION THREAD

**Discussion by ERNESTOGith** [community/community#69628] [Actions]
### Select Topic Area

General

### Body

Good evening

I have already installed Git and Github on my computer . But everytime I want to create a new repository, there is not the mention public or private, I don't know why.
What can I do?

**Comment by garousiamir** [comment]
I suggest you create your repo trough github website itself instead of the app. I had the same issue. after you created the repo in website you access that for ever in desktop app. you can clone it then in your pc and work as normal repo

**Comment by ERNESTOGith** [comment]
Thanks for your aid.
But, when I have tried to create a repo through the website by clicking on new, there was any page which opened. However, I expected to have a page where there is the mention 'public' and 'private'.

**Comment by vishalkrishna8** [comment]
Ensure you're logged in to your GitHub account. If not than visit the GitHub website (try to login by using website) , and than click on the "+" sign at the top right corner of the page, then select "New repository." and so on. 

If you still haven't found the solution to your problem than may be because of browser extensions or settings can affect how web pages are displayed. Try a different browser or disable extensions.

**Comment by ERNESTOGith** [comment]
Thanks for your aid,

When I used another browser (Chrome) as you had recommended me, I solved my problem. Previously, I used to use UC Browser.

**Reply by vishalkrishna8** [reply]
Glad that it was helpful to you, if you found my reply useful you can mark it as an answer☺ ,  Thank you.
